### PR TITLE
Document Waydroid

### PR DESCRIPTION
**Pull request by mirkobrombin**
This PR contributes by providing the steps to run Waydroid in a Distrobox. I explained the requirements and provided two installation methods: manual and automated (using a pre-configured image).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: docs/useful_tips.md [hunk 1]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
...
+```sh
+distrobox create --root \
+  --image ghcr.io/vanilla-os/pico:main \
+  --additional-packages "systemd libpam-systemd curl kmod dbus-x11 iptables mutter" \
+  --init \
+  --unshare-all \
+  --name waydroid
+```
+
+Once it's started with `distrobox enter --root waydroid`, we can proceed with the Waydroid
```
Comment (kbdharun) [inline docs/useful_tips.md:472]:
```suggestion
Once it's started with `distrobox enter --root waydroid`, we can proceed with the Waydroid
```

File: docs/useful_tips.md [hunk 2]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
...
+
+Waydroid is a popular solution for running Android applications on Linux using an LXC container. 
+Since these containers run inside a Distrobox, you can also run Waydroid.
+
+> **Note**: Wayland and the `binder_linux` module are required at the host level. You can install
+> the DKMS from the [choff/anbox-modules](https://github.com/choff/anbox-modules) repository.
+
+### Manual Installation
+
+To do this, we need a rootful container [with Systemd](#using-init-system-inside-a-distrobox) plus
```
Comment (kbdharun) [inline docs/useful_tips.md:451]:
```suggestion
To do this, we need a rootful container [with Systemd](#using-init-system-inside-a-distrobox) plus
```

File: docs/useful_tips.md [hunk 3]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
 
 And you have a working LXC inside your Distrobox container.
 
+## Using Waydroid inside a Distrobox
+
+Waydroid is a popular solution for running Android applications on Linux using an LXC container.
```
Comment (kbdharun) [inline docs/useful_tips.md:443]:
```suggestion
Waydroid is a popular solution for running Android applications on Linux using an LXC container.
```

File: docs/useful_tips.md [hunk 4]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
...
+export DBUS_SESSION_BUS_ADDRESS="unix:path=/run/host/$(echo "${DBUS_SESSION_BUS_ADDRESS}" | cut -d '=' -f2-)"
+
+waydroid init
+```
+
+The above environment variables must be present each time the `waydroid` command is used.
+
+### Automated Installation
+
+The [Waydroid image](https://github.com/Vanilla-OS/waydroid-image/blob/main/recipe.yml) from
```
Comment (kbdharun) [inline docs/useful_tips.md:495]:
```suggestion
The [Waydroid image](https://github.com/Vanilla-OS/waydroid-image/blob/main/recipe.yml) from
```

File: docs/useful_tips.md [hunk 5]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
...
+distrobox create --root \
+  --image ghcr.io/vanilla-os/waydroid:main \
+  --init \
+  --unshare-all \
+  --name waydroid
+
+distrobox enter --root waydroid
+```
+
+Once started, Waydroid is automatically executed via Systemd. Check for the process to finish using
```
Comment (kbdharun) [inline docs/useful_tips.md:508]:
```suggestion
Once started, Waydroid is automatically executed via Systemd. Check for the process to finish using
```

File: docs/useful_tips.md [hunk 6]
```diff
@@ -437,6 +438,83 @@ PID   USER     TIME  COMMAND
...
+```
+
+Once started, Waydroid is automatically executed via Systemd. Check for the process to finish using 
+the `systemctl status waydroid-init` command, then start using Waydroid with:
+
+```bash
+ewaydroid --help
+```
+
+Make sure to use the `ewaydroid` command each time you need to work with Waydroid. This command is a
```
Comment (kbdharun) [inline docs/useful_tips.md:515]:
```suggestion
Make sure to use the `ewaydroid` command each time you need to work with Waydroid. This command is a
```

**Review by kbdharun — COMMENTED** [review]
This would probably fix the markdown lint errors.

**Comment by 89luca89** [comment]
Great stuff thanks @mirkobrombin :heart:

**Pull request merged**